airflow/timetables/interval.py

```diff
@@ -232,7 +232,7 @@ def _skip_to_latest(self, earliest: Optional[DateTime]) -> DateTime:
             raise AssertionError("next schedule shouldn't be earlier")
         if earliest is None:
             return new_start
-        return max(new_start, earliest)
+        return max(new_start, self._get_next(earliest))
```

Comment:
This needs a test to ensure if `earliest` lies _exactly_ on the schedule, the first run is not one interval late. My intuition is this should use `self._align` instead, but a test would be required to verify that.

Reply:
should use anlign.  :cry: